docs/guides/misc/supporter_benefits.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: Supporter Model and Benefits
+---
+
+The [ComCODE Supporter Model](https://givebutter.com/QncoAxxT2tdWf3UfJ2x95z25n1HNizxI) intends to provide a way for anyone to support ComCODE's long term development of hackmud.
+
+By supporting [ComCODE](https://comcode.org/) you are making a donation to ComCODE, a 501(c)-3 Non-Profit, on a monthly recurring basis. This will help us to achieve our goal of financial sustainability, so that we can continue to deliver on our mission.
```

Comment:
```suggestion
By supporting [ComCODE](https://comcode.org/) you are making a donation to ComCODE, a 501(c)(3) nonprofit organization, on a monthly recurring basis. This will help us to achieve our goal of financial sustainability, so that we can continue to deliver on our mission.
```